UnitTests/GitCommandsTests/Git/Gpg/GitGpgControllerTests.cs

```diff
@@ -33,7 +33,7 @@ public void Setup()
         [TestCase(CommitStatus.NoSignature, "N")]
         public async Task Validate_GetRevisionCommitSignatureStatusAsync(CommitStatus expected, string gitCmdReturn)
         {
-            var guid = Guid.NewGuid().ToString("N");
+            var guid = ObjectId.Random();
```

Comment:
Added a `Random` factory method for use in unit tests.

Reply:
If `Random` is used only in unit tests then it shouldn't be added to the production code.
Please move this behaviour into the `CommonTestUtils` test project